src/odemis/acq/test/acq_test.py

```diff
@@ -324,21 +314,11 @@ def setUpClass(cls):
         cls.spec_det_sel = model.getComponent(role="spec-det-selector")
         cls.ar_spec_sel = model.getComponent(role="ar-spec-selector")
 
-    @classmethod
```

Comment:
Why is the teardown removed?

Reply:
That's a little bit unrelated, but since a couple of years, `start_backend()` is more clever and can automatically stop the backend if the wrong one is in use, and start the right one. So we don't need all this code to dance around the possibility that the wrong backend is used. It especially saves time when you run test cases multiple times in a row, as the backend isn't restarted every time.
Most of the test cases have been updated already, but this one wasn't.